docs/changelog.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+The changelog documents all notable changes to the project. This file is updated regularly with details about new features, bug fixes, and other changes.
+
+## [Unreleased]
+
+- List any upcoming changes here.
+
+## [Version] - [Release Date]
+
+- List changes for each version.
```

Comment:
We don't release metamist in named versions, we release on every push to main (which as a byproduct often stages a couple of changes together). Definitely happy for a running list of metamist changes, but as we squash merge, the commit history of `dev` does a reasonable job of that.

We don't have (AFAIK) any consumers that actually watch for generated python API changes, they usually do this from the analysis-runner / driver image which we try to keep consistently up to date to avoid any out-of-sync issues. I'm definitely not against this, I'm not confident that people will use it unless we make a bigger deal out of it.

Thoughts?

Reply:
Awesome! Was just as a placeholder in case we wanted to add but I didn't really see a use for it either, hence I left it blank. I'll remove this :))